src/backend/types.ml

```diff
@@ -137,11 +140,11 @@ module Division = struct
       ~items:(
         obj @@ S.make_schema ()
           ~typ:string
-          ~enum:[
-            `String "Novice";
-            `String "Intermediate";
-            `String "Advanced";
-          ])
+          ~enum:(List.map
+                   (fun division -> `String (show division))
+                   all
+                )
```

Comment:
I think this code is more complex and less reliable than the old code (you replaced 3 lines of easily reviewed code with more than one ppx invocation and code that need a lot of invariants on the code generated by the ppxes).